src/wasm/context.rs

```diff
@@ -0,0 +1,23 @@
+use crate::Event;
+use std::collections::LinkedList;
+
+#[derive(Default)]
+pub(super) struct EventBuffer {
+    pub(super) events: LinkedList<Event>,
```

Comment:
Just curious, why a linked list?

Reply:
Right now we're only emitting 0 or 1 events per `process` so it was do a linkedlist or preallocate a vector with some capacity. Eventually I want to make it so we can do 0...n so a linkedlist is actually appropriate here, especially if we end up having the list feed to a stream which may only take some m events.

Reply:
Yeah I mostly meant why linked list instead of a vector, but I'm sure it's not a big deal either way at this point.